src/makers/finestra/finestra.svg

```diff
@@ -1,40 +1 @@
-	<path id="Rectangle" class="st0" d="M16.7,2h33.2c7.4,0,13.3,6,13.3,13.3v33.6c0,7.4-6,13.3-13.3,13.3H16.7
-		c-7.4,0-13.3-6-13.3-13.3V15.3C3.4,7.9,9.4,2,16.7,2z"/>
-</g>
-<g class="st1">
-	<path class="st2" d="M22.5,37.5c0-0.6,0.2-1.2,0.7-1.6c0.5-0.5,1-0.7,1.6-0.7v1c-0.4,0-0.7,0.1-0.9,0.4c-0.2,0.2-0.3,0.4-0.3,0.6
-		h1.3v1h-1.3v3h-1V37.5z"/>
-	<path class="st2" d="M26.1,36.2c-0.1,0-0.3,0-0.4-0.1s-0.1-0.2-0.1-0.4c0-0.1,0-0.3,0.1-0.4s0.2-0.1,0.4-0.1c0.1,0,0.3,0,0.4,0.1
-		s0.1,0.2,0.1,0.4c0,0.1,0,0.3-0.1,0.4S26.2,36.2,26.1,36.2z M26.6,36.5v4.6h-1v-4.6H26.6z"/>
-	<path class="st2" d="M28.3,41.2h-1v-2.3c0-0.6,0.2-1.2,0.7-1.6c0.5-0.5,1-0.7,1.6-0.7c0.6,0,1.2,0.2,1.6,0.7c0.5,0.5,0.7,1,0.7,1.6
-		v2.3h-1v-2.3c0-0.4-0.1-0.7-0.4-0.9s-0.6-0.4-0.9-0.4s-0.7,0.1-0.9,0.4s-0.4,0.6-0.4,0.9V41.2z"/>
-	<path class="st2" d="M34,39.8c0.3,0.3,0.6,0.4,0.9,0.4s0.7-0.1,0.9-0.4l0,0l0.7,0.7l0,0c-0.5,0.5-1,0.7-1.6,0.7
-		c-0.6,0-1.2-0.2-1.6-0.7c-0.5-0.5-0.7-1-0.7-1.6c0-0.6,0.2-1.2,0.7-1.6c0.5-0.5,1-0.7,1.6-0.7c0.6,0,1.2,0.2,1.6,0.7
-		c0.5,0.5,0.7,1,0.7,1.6c0,0.1,0,0.2,0,0.3h-0.3h-3.3C33.7,39.4,33.8,39.6,34,39.8z M35.9,37.9c-0.3-0.3-0.6-0.4-0.9-0.4
-		s-0.7,0.1-0.9,0.4c-0.1,0.1-0.2,0.2-0.2,0.3h2.3C36,38.1,36,38,35.9,37.9z"/>
-	<path class="st2" d="M39.3,36.5h2v1h-1.8c-0.4,0-0.5,0.1-0.5,0.2c0,0.2,0.4,0.4,1.1,0.6c0.5,0.1,0.9,0.3,1.1,0.5
-		c0.2,0.2,0.3,0.5,0.3,0.9c0,0.5-0.1,0.9-0.4,1.1s-0.7,0.4-1.3,0.4H38l0-1h1.8c0.4,0,0.7-0.1,0.7-0.3c0-0.2-0.1-0.3-0.4-0.4
-		c-1-0.2-1.5-0.5-1.7-0.7c-0.2-0.2-0.3-0.5-0.3-0.9C38,37,38.4,36.5,39.3,36.5z"/>
-	<path class="st2" d="M43,37.5v1.3c0,0.4,0.1,0.7,0.4,0.9s0.6,0.4,0.9,0.4v1c-0.6,0-1.2-0.2-1.6-0.7c-0.5-0.5-0.7-1-0.7-1.6v-3.6h1
-		v1.3h1.3v1H43z"/>
-	<path class="st2" d="M45,38.9c0-0.6,0.2-1.2,0.7-1.6c0.5-0.5,1-0.7,1.6-0.7v1c-0.4,0-0.7,0.1-0.9,0.4S46,38.5,46,38.9v2.3h-1V38.9z
-		"/>
-	<path class="st2" d="M52.7,38.9c0,0.6-0.2,1.2-0.7,1.6s-1,0.7-1.6,0.7c-0.6,0-1.2-0.2-1.6-0.7c-0.5-0.5-0.7-1-0.7-1.6
-		c0-0.6,0.2-1.2,0.7-1.6c0.5-0.5,1-0.7,1.6-0.7c0.5,0,0.9,0.1,1.3,0.4v-0.4h1V38.9z M51.3,39.8c0.3-0.3,0.4-0.6,0.4-0.9
-		c0-0.4-0.1-0.7-0.4-0.9s-0.6-0.4-0.9-0.4c-0.4,0-0.7,0.1-0.9,0.4S49,38.5,49,38.9c0,0.4,0.1,0.7,0.4,0.9s0.6,0.4,0.9,0.4
-		C50.7,40.2,51,40.1,51.3,39.8z"/>
-</g>
-<rect x="3" y="2.7" class="st3" width="69.2" height="70.4"/>
-</svg>
+<svg id="katman_1" data-name="katman 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 801.49 807.63"><defs><style>.cls-1{fill:none;stroke:#000;stroke-miterlimit:10;stroke-width:18.98px;}</style></defs><rect class="cls-1" x="9.49" y="9.49" width="782.51" height="788.65" rx="204.49"/><path d="M301.08,293.35H503.21a0,0,0,0,1,0,0v18A36.18,36.18,0,0,1,467,347.53H301.08a0,0,0,0,1,0,0V293.35A0,0,0,0,1,301.08,293.35Z"/><path d="M301.08,374H427.74a0,0,0,0,1,0,0v18a36.18,36.18,0,0,1-36.18,36.18H301.08a0,0,0,0,1,0,0V374A0,0,0,0,1,301.08,374Z"/><path d="M301.08,454.74h50.68a0,0,0,0,1,0,0v20.34a33.84,33.84,0,0,1-33.84,33.84H301.08a0,0,0,0,1,0,0V454.74A0,0,0,0,1,301.08,454.74Z"/></svg>
```

Comment:
Your path needs to have a fill color like this
```
<svg id="katman_1" data-name="katman 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 801.49 807.63">
  <defs>
    <style>.cls-1{fill:none;stroke:#000;stroke-miterlimit:10;stroke-width:18.98px;}</style>
  </defs>
  <rect class="cls-1" x="9.49" y="9.49" width="782.51" height="788.65" rx="204.49" />
  <path fill="#000" d="M301.08,293.35H503.21a0,0,0,0,1,0,0v18A36.18,36.18,0,0,1,467,347.53H301.08a0,0,0,0,1,0,0V293.35A0,0,0,0,1,301.08,293.35Z" />
  <path fill="#000" d="M301.08,374H427.74a0,0,0,0,1,0,0v18a36.18,36.18,0,0,1-36.18,36.18H301.08a0,0,0,0,1,0,0V374A0,0,0,0,1,301.08,374Z" />
  <path fill="#000" d="M301.08,454.74h50.68a0,0,0,0,1,0,0v20.34a33.84,33.84,0,0,1-33.84,33.84H301.08a0,0,0,0,1,0,0V454.74A0,0,0,0,1,301.08,454.74Z" />
</svg>
```

Reply:
Thx Matthew, I added svg path fill color in the new commit.